### ISSUE TITLE
Track queue time for a Sidekiq worker

### ISSUE THREAD

**Issue by tombruijn** [appsignal/appsignal-ruby#494]
Track queue time per Sidekiq worker so we can show a graph on our magic Sidekiq dashboard about it. This so you can differentiate between workers in the whole namespace.

- [ ] Add metric to SidekiqPlugin: https://github.com/appsignal/appsignal-ruby/blob/01cf98491b539c94d32d651bb774a340f946a0e3/lib/appsignal/hooks/sidekiq.rb#L99
- [ ] Update Sidekiq dashboard: https://github.com/appsignal/public_config/blob/d83984e1092fa39d413ee0c8d2eb2d28a3058775/dashboards/sidekiq/main.yml

Per request: https://app.intercom.io/a/apps/yzor8gyw/inbox/inbox/540709/conversations/21372152283

**Comment by lucasmazza** [comment]
This was once implemented on appsignal/appsignal-ruby#112 but looks like #362 remove it 😬

**Comment by tombruijn** [comment]
Hi @lucasmazza, not sure what you mean. This issue is about tracking it per worker, not overall queue time. Is the overal queue time for Sidekiq broken for you?

**Comment by thijssondag** [comment]
Yes, would be great to track it per worker!

**Comment by tombruijn** [comment]
> This was once implemented on #112 but looks like #362 remove it 😬

I don't think this worked. I don't know why [our custom background job instrumentation docs](https://docs.appsignal.com/ruby/instrumentation/background-jobs.html) mention a `queue_time` option. No component, Ruby gem, extension, agent, or processor read from that option.
I'm going to update it to mention `:queue_start`, which it does listens to.
